src/api/ledger/transferUsingVaultIdAndStarkKey.js

```diff
@@ -1,9 +1,19 @@
 const { post } = require('request-promise')
 const makeCreateSignedTransferTxLedger = require('../../lib/ledger/makeCreateSignedTransferTxLedger')
 
-module.exports = async (dvf, transferData, path, feeRecipient) => {
+module.exports = async (
+  dvf,
+  transferData,
+  path,
+  feeRecipient,
+  walletSignCb = () => {}
+) => {
   const url = dvf.config.api + '/v1/trading/w/transfer'
   const createSignedTransferTx = makeCreateSignedTransferTxLedger(dvf)(path)
-  const json = await dvf.createTransferPayload(transferData, feeRecipient, createSignedTransferTx)
-  return post(url, { json })
+  const json = await dvf.createTransferPayload(
+    transferData,
+    feeRecipient,
+    createSignedTransferTx
+  )
+  return post(url, { json }, walletSignCb)
```

Comment:
I think technically you can say that the user has signed after `dvf.createTransferPayload` is successful (maybe you want to call you callback at this post ?)
The `post` is just the HTTP request, which as far as I understand will resolve by itself when successful (not sure how `walletSignCb` plays out passed like this)

Reply:
Yes you are completely right, I'll make the change, thanks